scripts/ci/docker-compose/integration-keycloak.yml

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+services:
+  keycloak:
+    image: quay.io/keycloak/keycloak:23.0.6
+    labels:
+      breeze.description: "Integration for manual testing multi-team Airflow."
+    entrypoint: /opt/keycloak/keycloak-entrypoint.sh
+    environment:
+      KC_HOSTNAME: localhost
+      KC_HOSTNAME_PORT: 38080
+      KC_HOSTNAME_STRICT_BACKCHANNEL: false
+      KC_HTTP_ENABLED: true
+      KC_HOSTNAME_STRICT: true
+
+      KEYCLOAK_ADMIN: admin
+      KEYCLOAK_ADMIN_PASSWORD: admin
+
+      KC_DB: postgres
+      KC_DB_URL: jdbc:postgresql://postgres/keycloak
+      KC_DB_USERNAME: keycloak
+      KC_DB_PASSWORD: keycloak
+    ports:
+      - 38080:38080
+    restart: always
+    depends_on:
+      - postgres
+    volumes:
+      - ./keycloak/keycloak-entrypoint.sh:/opt/keycloak/keycloak-entrypoint.sh
+
+  postgres:
```

Comment:
I am not sure we should use the same postgres service here as for the main airflow. The `postgres-backend.yml` will only be used when `--backend postgres` is used, but in other cases it won't be. 

I'd say this postgres instance should be completely independent - and named differently - becuase then you can also restart and delete any volumes for it independently from Airlfow DB.

Reply:
Hmmm, those are fair points, but definitely complicates things. What if we just make the dependency required (in logic in Breeze code) that --integration keycloak _must_ be used in conjunction with --backend postgres? I don't think usage of this integration will be very high, so I don't think this is unreasonable to keep the amount of docker compose code we're managing small. WDYT?

Reply:
That's also an option.